### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by arthurdm**
Once the [non-root PR](https://github.com/docker-library/official-images/pull/4959) goes through, we can merge this one for the related documentation.

### CODE REVIEW

File: websphere-liberty/content.md [hunk 1]
```diff
@@ -2,14 +2,25 @@
 
 The images in this repository contain IBM WebSphere Application Server Liberty for Developers and the IBM Java Runtime Environment. See the license section below for restrictions relating to the use of this image. For more information about WebSphere Application Server Liberty, see the [WASdev](https://developer.ibm.com/wasdev/docs/category/getting-started/) site.
 
-# Images
+## Image User
 
-There are multiple images available in this repository. The image with the tag `beta` contains the contents of the install archive for the latest monthly beta. The other images are all based on the latest generally available fix pack.
+This image runs by default with `USER 1001` (non-root), as part of `group 0`.   All of the folders accessed by WebSphere Liberty been given the appropriate permission, but if your extending Dockerfile needs permission to another location you can simply temporarily switch into root and provide the needed permissions, example:
+
+```
```
Comment (yosifkit) [inline websphere-liberty/content.md]:
```suggestion
```dockerfile
```

Just need a code block syntax to make `markdownfmt` happy.

**Comment by arthurdm** [comment]
thanks for the update @yosifkit!

File: websphere-liberty/content.md [hunk 2]
```diff
@@ -2,14 +2,25 @@
 
 The images in this repository contain IBM WebSphere Application Server Liberty for Developers and the IBM Java Runtime Environment. See the license section below for restrictions relating to the use of this image. For more information about WebSphere Application Server Liberty, see the [WASdev](https://developer.ibm.com/wasdev/docs/category/getting-started/) site.
 
-# Images
+## Image User
 
-There are multiple images available in this repository. The image with the tag `beta` contains the contents of the install archive for the latest monthly beta. The other images are all based on the latest generally available fix pack.
+This image runs by default with `USER 1001` (non-root), as part of `group 0`.   All of the folders accessed by WebSphere Liberty been given the appropriate permission, but if your extending Dockerfile needs permission to another location you can simply temporarily switch into root and provide the needed permissions, example:
```
Comment (tianon) [inline websphere-liberty/content.md]:
```suggestion
This image runs by default with `USER 1001` (non-root), as part of `group 0`. All of the folders accessed by WebSphere Liberty been given the appropriate permission, but if your extending Dockerfile needs permission to another location you can simply temporarily switch into root and provide the needed permissions, example:
```

(There's a few extra "` `"s here that `markdownfmt` is still sad about.)

Comment (yosifkit) [reply websphere-liberty/content.md]: :cry: it won't let me apply the suggestion. :confused:

Comment (arthurdm) [reply websphere-liberty/content.md]: applied =)

**Comment by arthurdm** [comment]
thx guys.  suggestions applied (I like this new feature!)

**Comment by arthurdm** [comment]
@tianon @yosifkit - can we please merge this one?      customers will be wondering about the non-root change.  thx!

**Comment by tianon** [comment]
Doh, sorry!  Merging now!

**Pull request merged**